go.mod

```diff
@@ -97,14 +97,13 @@ require (
 	k8s.io/client-go v0.21.0
 	k8s.io/cluster-bootstrap v0.20.5
 	k8s.io/helm v2.14.3+incompatible
-	k8s.io/kube-openapi v0.0.0-20210323165736-1a6458611d18 // indirect
 	k8s.io/kubelet v0.0.0
 	k8s.io/kubernetes v1.20.5
 	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10
 	rsc.io/letsencrypt v0.0.3 // indirect
 	sigs.k8s.io/application v0.8.3
 	sigs.k8s.io/controller-runtime v0.8.3
-	sigs.k8s.io/kustomize/api v0.8.5
+	sigs.k8s.io/kustomize/api v0.3.3
```

Comment:
Had to downgrade this because kustomize v3.5.4 uses this version of the api

Reply:
dependabot is not gonna be happy about this :)

Reply:
reverted this to v0.8.5 since the replace is sufficient